content/en/network_performance_monitoring/devices/guide/tags-with-regex.md

```diff
@@ -0,0 +1,69 @@
+---
+title: NDM Tags with Regex
+kind: guide
+further_reading:
+- link: "/network_performance_monitoring/devices/setup"
+  tag: "Documentation"
+  text: "Network Device Monitoring Setup"
+- link: "/getting_started/tagging"
+  tag: "Documentation"
+  text: "Getting Started with Tags"
+---
+
+Datadog Network Device Monitoring (NDM) supports regular expressions to create metric tags in the format `<KEY>:<VALUE>`.
+
+## Setup
+
+### Installation
+
+Follow the [setup instructions][1] to install Datadog NDM.
+
+### Configuration
+
+In the [SNMP conf.yaml][2], you can specify `metric_tags` from an OID. To create multiple tags for devices, use regular expressions to separate the resulting value into multiple tags, or get a substring using the regular [Python engine][3].
+
+#### OID
+
+The example below creates two tags using Regex matching on the OID's value. So, if the OID's value is `41ba948911b9`, the tags `host_prefix:41` and `host:ba948911b9` are added to the corresponding metrics.
```

Comment:
Do we have an official stance on how we say "regex" in the docs? I did a quick search just in the documentation repo and it looks like it's often lowercase.